amundsen_application/static/css/_layouts.scss

```diff
@@ -94,11 +96,11 @@ $resource-header-height: 84px;
       flex-basis: 600px;
       flex-shrink: 0;
       overflow-y: scroll;
-      padding: 0 24px;
+      padding: $aside-separation-space 24px;
 
       > .banner {
         border: 1px solid $stroke;
-        height: 40px;
+        height: $aside-separation-space;
```

Comment:
nit: I think it's a coincidence that the spacing and banner height are the same value. I'd feel better making this it's own variable, because design could let's say change the spacing but not change the banner.

Reply:
OK, I will rename it.